code/modules/projectiles/guns/projectile/battle_rifle/boltgun.dm

```diff
@@ -176,7 +176,6 @@
 	w_class = ITEM_SIZE_HUGE
 	slot_flags = SLOT_BACK
 	damage_multiplier = 1.2
-	penetration_multiplier = 1.3
```

Comment:
is this an intended removal?

Reply:
LE EPIC STEALTH NERF

Reply:
Balance PR 7116: Sneaky Edition

Reply:
how did that get there

Reply:
Congrats, The Curse has been transferred to you.

Reply:
Should be fixed